etc/generate-envoy-config/envoy.libsonnet

```diff
@@ -0,0 +1,201 @@
+{
+  bootstrap(listeners, clusters): {
+    admin: {
+      access_log: [
+        {
+          name: 'envoy.access_loggers.stderr',
+          typed_config: {
+            '@type': 'type.googleapis.com/envoy.extensions.access_loggers.stream.v3.StderrAccessLog',
+          },
+        },
+      ],
+      address: {
+        socket_address: {
+          address: '0.0.0.0',
+          port_value: 9901,
```

Comment:
which port is this?

Reply:
Envoy admin interface.  Used for the k8s health check (/server_info = live, /ready = ready).  Also serves /stats/prometheus for prometheus enthusiasts.